server/chat-plugins/info.js

```diff
@@ -1530,13 +1532,21 @@ const commands = {
 		if (cmd === 'calc' && target) return this.parse(`/math ${target}`);
 		if (!this.runBroadcast()) return;
 		const SUPPORTED_RANDOM_FORMATS = ['gen7randombattle', 'gen7unratedrandombattle'];
+		const SUPPORTED_BATTLESPOT_FORMATS = ['gen5gbusingles', 'gen5gbudoubles', 'gen6battlespotsingles', 'gen6battlespotdoubles', 'gen6battlespottriples', 'gen7battlespotsingles', 'gen7battlespotdoubles', 'gen7bssfactory'];
 		const isRandomBattle = (room && room.battle && SUPPORTED_RANDOM_FORMATS.includes(room.battle.format));
+		const isBattleSpotBattle = (room && room.battle && (SUPPORTED_BATTLESPOT_FORMATS.includes(room.battle.format) || room.battle.format.includes("battlespotspecial")));
 		if (['randomscalc', 'randbatscalc', 'rcalc'].includes(cmd) || isRandomBattle) {
 			return this.sendReplyBox(
 				`Random Battles damage calculator. (Courtesy of LegoFigure11 &amp; Smoochyena)<br />` +
 				`- <a href="https://randbatscalc.github.io/">Random Battles Damage Calculator</a>`
 			);
 		}
+		if (['bsscalc', 'cantsaycalc'].includes(cmd) || isBattleSpotBattle) {
```

Comment:
Commands aren't case-sensitive so this will return the Honko damage calc if someone used `/Bsscalc` in a non battle spot battle. 
```suggestion
		if (['bsscalc', 'cantsaycalc'].includes(cmd.toLowerCase()) || isBattleSpotBattle) {
```

Reply:
`cmd` is already passed as lowecase, so ignore this review as well as the one below this, my bad.